scripts/us_census/pep/pep_by_asrh/preprocess.py

```diff
@@ -0,0 +1,945 @@
+        '3': "Male_HispanicOrLatino_White",
+        '4': "Female_HispanicOrLatino_White",
+        '5': "Male_Black",
+        '6': "Black_Female",
+        '7': "Male_AmericanIndianAndAlaskaNativeAlone",
+        '8': "Female_AmericanIndianAndAlaskaNativeAlone",
+        '9': "Male_AsianOrPacificIslander",
+        '10': "Female_AsianOrPacificIslander",
+        '11': "Male_HispanicOrLatino",
+        '12': "Female_HispanicOrLatino"
+    }
+    df["SV"] = df["SV"].map(sv_dict)
+    # Deriving New Columns
+    derived_df = pd.DataFrame()
+    derived_cols = _county_1900_1999()
+    for dsv, sv in derived_cols.items():
+        flag = True
+        for col in sv:
+            derived_tmp_df = df[df["SV"] == col].reset_index(drop=True)
+            if derived_df.shape[0] == 0:
+                derived_df = derived_tmp_df
+            else:
+                derived_df[pop_cols] += derived_tmp_df[pop_cols]
+        if flag:
+            derived_df["SV"] = dsv
+            df = df.append(derived_df)
+    df = df.dropna()
+    df = _transpose_df(df, ["Year", "Location", "SV"],
+                       pop_cols,
+                       default_col="Age")
+    # Creating SV's name using SV, Age Column
+    df["SV"] = df.apply(lambda row: _create_sv(row.SV, row.Age), axis=1)
+    # Creating GeoId's for State FIPS Code
+    df["Location"] = "geoId/" + df["Location"].astype("str").str.pad(
+        width=5, side="left", fillchar="0")
+    # Deriving Measurement Method for the SV's
+    df = _add_measurement_method(df, "SV", "Measurement_Method")
+    df = df[["Year", "Location", "SV", "Measurement_Method", "Count_Person"]]
+    return df
+
+
+def _process_county_2000_2009(file_path: str) -> pd.DataFrame:
+    """
+    Returns the Cleaned DataFrame consists
+    county data for the year 2000-2009
+
+    Args:
+        file_path (str): Input File Path
+
+    Returns:
+        pd.DataFrame: Cleaned DataFrame
+    """
+    df = _load_df(file_path, "csv", header=0, encoding="latin-1")
+    df = df[(~df["YEAR"].isin([1, 12, 13])) &
+            (df["AGEGRP"] != 99)].reset_index(drop=True)
+    df["YEAR"] = 1998 + df["YEAR"]
+
+    cols = list(df.columns)
+    # Mapping Dataset Headers to its FullForm
+    cols_dict = get_cols_dict()
+    for idx, val in enumerate(cols):
+        cols[idx] = cols_dict.get(val, val)
+    df.columns = cols
+    df["STATE"] = df["STATE"].astype("str").str.pad(width=2,
+                                                    side="left",
+                                                    fillchar="0")
+    df["COUNTY"] = df["COUNTY"].astype("str").str.pad(width=3,
+                                                      side="left",
+                                                      fillchar="0")
+    df["Location"] = "geoId/" + df["STATE"] + df["COUNTY"]
+    # Deriving New Columns
+    derived_cols = _county_2000_2009()
+    for dsv, sv in derived_cols.items():
+        df[dsv] = df.loc[:, sv].sum(axis=1)
+        cols.append(dsv)
```

Comment:
I see this code block used twice and similarly for other blocks as well. Could we re-use as functions?